service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/StandardDirectionsOrderIssuedEventHandlerTest.java

```diff
@@ -147,27 +135,18 @@ void shouldNotifyLocalAuthorityOfIssuedSDOWhenSendNoticeOfProceedingsIsDisabled(
         given(inboxLookupService.getRecipients(caseData))
             .willReturn(Set.of(LOCAL_AUTHORITY_EMAIL_ADDRESS));
 
-        given(featureToggleService.isSendNoticeOfProceedingsFromSdo())
```

Comment:
is test name still correct?

Reply:
removing test not needed as toggle on / off one are both there